### PR TITLE
deploy triage-party #1649 ArgoCD Deployment Manifests

### PR DESCRIPTION
**Pull request by Gregory-Pereira**
Added prod and test ArgoCD manifests for triage party in MOC Zero environment.

Related-to: https://github.com/thoth-station/thoth-application/issues/1649

### CODE REVIEW

File: envs/moc/zero/thoth/test/test-thoth-triage-party.yaml [hunk 1]
```diff
@@ -0,0 +1,26 @@
+---
```
Comment (harshad16) [inline envs/moc/zero/thoth/test/test-thoth-triage-party.yaml:1]: we just need one deployment of the app.
you can drop this one.

File: envs/moc/zero/thoth/prod/prod-thoth-triage-party.yaml [hunk 2]
```diff
@@ -0,0 +1,26 @@
+---
```
Comment (harshad16) [inline envs/moc/zero/thoth/prod/prod-thoth-triage-party.yaml:1]: as this is application object file, and we would like the argocd deployment to consider this as well
we have to update the Kustomize file. 
update the name of file in here as well, https://github.com/operate-first/argocd-apps/blob/main/envs/moc/zero/thoth/prod/kustomization.yaml

**Review by harshad16 — DISMISSED** [review]
Very well done :100: 
just few minor comments.

**Review by tumido — COMMENTED** [review]
/lgtm

good job :+1:

**Review by harshad16 — COMMENTED** [review]
/lgtm
very well done :100:

**Review by tumido — APPROVED** [review]
/approve

**Pull request merged**